txtorcon/torconfig.py

```diff
@@ -10,8 +10,11 @@
 import functools
 import warnings
 from io import StringIO
-from collections import OrderedDict
 from warnings import warn
+try:
+    from collections.abc import OrderedDict
```

Comment:
To my best knowledge, this will never work. OrderedDict is not an abstract base class.